src/mlpack/methods/ann/layer/instance_norm.hpp

```diff
@@ -0,0 +1,214 @@
+/**
+ * @file methods/ann/layer/instance_norm.hpp
+ * @author Anjishnu Mukherjee
```

Comment:
I think this might need to be updated, shouldn't you add your name here too @hello-fri-end?

Reply:
Agreed, feel free to add you as an author.